src/main/java/pazone/ashot/coordinates/WebDriverCoordsProvider.java

```diff
@@ -1,20 +1,25 @@
 package pazone.ashot.coordinates;
 
-import org.openqa.selenium.Rectangle;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 
+
```

Comment:
useless empty line

Reply:
Sorry, my IDE sometimes does this. Gonna fix it.